dashboard/src/ui/dashboard/components/UsageLimitsPanel.jsx

```
@@ -642,6 +657,65 @@ function OpenCodeGoSetupHint() {
+    try {
+      await save(payload);
+      setSaveState("saved");
+      setSaveError("");
+      setApiKey("");
+      setAuthCookie("");
+      setWorkspaceId("");
+      // Bust the 2-minute usage-limits cache so the next Limits fetch
+      // doesn't flash the previous error (see src/lib/usage-limits.js CACHE_TTL_MS).
+      try {
+        await fetch("/functions/tokentracker-usage-limits?refresh=1", { cache: "no-store" });
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)